app/src/main/java/com/example/mystable/model/Data.kt

```diff
@@ -0,0 +1,72 @@
+package com.example.mystable.model
+
+import com.example.mystable.pojo.TabDetails
+import com.example.mystable.pojo.TabInfo
+import com.example.mystable.pojo.TabItems
+
```

Comment:
Don’t use misleading class names. You should use a clear name for what a class should do like: “MarketPlaceDataSource” Instead of Data

Reply:
👍